src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -114,7 +114,13 @@ public class ProxyOperationManager
     /// <summary>
     /// Gets the proxy operation manager id for proxy test session manager internal organization.
     /// </summary>
-    public int Id { get; set; } = -1;
+    internal int Id { get; set; } = -1;
```

Comment:
Indeed, this `Id` is only used in the context of test sessions, which right now is a workflow that goes unused. The `Id` got added in the initial release of test sessions though, not now, but we should be safe since nobody uses the APIs yet.

Reply:
but is it used somewhere? if not I'd remove